### PR TITLE
add arg to create_image to set output filename

### PR DESCRIPTION
**Pull request by dougfelt**

### CODE REVIEW

File: nototools/create_image.py [hunk 1]
```diff
@@ -20,9 +20,10 @@
 __author__ = 'roozbeh@google.com (Roozbeh Pournader)'
 
 import argparse
+import codecs
```
Comment (jamesgk) [inline nototools/create_image.py]: Is this still used?

File: nototools/create_image.py [hunk 2]
```diff
@@ -198,29 +208,34 @@ def _get_style(style_name):
...
+      name_strs = [font.replace(' ', '')]
+      name_strs.extend(['%x' % ord(cp) for cp in text])
+      if weight_name:
+        name_strs.append(weight_name)
+      if style_name:
+        name_strs.append(style_name)
+      name_strs.append(str(font_size))
+      if lang:
+        name_strs.append(lang)
+        file_name = '_'.join(name_strs) + '.' + ext
```
Comment (jamesgk) [inline nototools/create_image.py]: Should this line be un-indented?

File: nototools/create_image.py (hunk 1 above)
Comment (dougfelt) [inline nototools/create_image.py]: I removed the import from the other location.  My java bias leads me to prefer putting the imports at the top of the file, even it if means we import some things we don't use unless we make a particular call.

File: nototools/create_image.py (hunk 2 above)
Comment (dougfelt) [inline nototools/create_image.py]: Sure should, thanks.

File: nototools/create_image.py (hunk 1 above)
Comment (jamesgk) [inline nototools/create_image.py]: Oh, I don't think it was actually removed in this commit.

**Comment by dougfelt** [comment]
sorry, replied before I'd pushed.  now the push is up.

**Comment by jamesgk** [comment]
LGTM

**Pull request merged**